client/src/actions/notificationActionCreator.jsx

```diff
@@ -0,0 +1,27 @@
+export function fetchNotifications(bool) {
+  return {
+    type: 'FETCHING',
+    bool
```

Comment:
Sorry, Winston. I guess it's better to name the payload on line 4 as isFetching..... otherwise bool is a variable and you'll have difficulty accessing it in the reducer... So sorry